src/main/kotlin/sort/BestNumber.kt

```diff
@@ -0,0 +1,37 @@
+package sort
+
+fun main() {
+    println(solution(intArrayOf(3, 30, 34, 5, 9)))
+}
+
+fun solution(numbers: IntArray): String {
+    val comparator = Comparator<String> { o1, o2 ->
+        (o2 + o1).compareTo(o1 + o2)
+    }
+    if(numbers.all { it ==0 }) return "0"
```

Comment:
```suggestion
    if(numbers.all { it == 0 }) return "0"
```

코테도 정렬을 신경써봅시당!
all 함수는 첨보는데 배열의 모든 원소가 주어진 조건을 만족하는지 검사하는 친구군용! 유용하게 사용할 수 있겠네요 ㅎㅎ

Reply:
compareTo()를 코테 문제풀이 보면서 자주 활용되는 친구같군요~! good!! 오늘도 배워갑니다~